javacutil/src/main/java/org/checkerframework/javacutil/SystemUtil.java

```diff
@@ -244,8 +246,10 @@ public static String conjunction(String conjunction, List<?> elements) {
      * @param c a collection
      * @return a list of the results of applying {@code f} to the elements of {@code list}
      */
-    public static <FROM, TO> List<TO> mapList(
-            Function<? super FROM, ? extends TO> f, Collection<FROM> c) {
+    public static <
+                    @KeyForBottom FROM extends @UnknownKeyFor Object,
+                    @KeyForBottom TO extends @UnknownKeyFor Object>
+            List<TO> mapList(Function<? super FROM, ? extends TO> f, Collection<FROM> c) {
         return c.stream().map(f).collect(Collectors.toList());
```

Comment:
This is not the implementation in plume-util.  (That implementation changed and I thought you were going to pick up the new implementation.)